entities/library.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+class Library
+  include Validation
+  include Statistics
+
+  attr_accessor :authors, :readers, :books, :orders
+
+  def initialize
+    @authors = []
+    @books = []
+    @readers = []
+    @orders = []
+
+    load_datas
+  end
+
+  def load_datas
+    @authors = YAML.load_file(AUTHORS_DB) if File.exist?(AUTHORS_DB)
+    @readers = YAML.load_file(READERS_DB) if File.exist?(READERS_DB)
+    @books = YAML.load_file(BOOKS_DB) if File.exist?(BOOKS_DB)
+    @orders = YAML.load_file(ORDERS_DB) if File.exist?(ORDERS_DB)
+  end
+
+  def save
+    File.open(AUTHORS_DB, 'w') { |file| file.write(@authors.to_yaml) }
```

Comment:
move File.open(AUTHORS_DB, 'w') { |file| file.write(@authors.to_yaml) } to separate method and pass symbol to it